### DISCUSSION TITLE
how to return an api request response header using js in the transfomer

### DISCUSSION THREAD

**Discussion by MassoundiOmar** [Budibase/budibase#12278] [General]
Hi all
I m looking for a way to return my set-cookie header using js in the transfomer? Is it possible ? if someone have a script exemple... It would be very helpful... Thank you

<img width="508" alt="header" src="https://github.com/Budibase/budibase/assets/97040347/c3509a8f-7387-43c4-9007-a65805418e6c">

**Comment by melohagan** [comment]
Hey @MassoundiOmar 

The transformer is specific to the response body to be consumed by Budibase components, and has no awareness of headers.
